### PR TITLE
New company suggestion: Slovnaft Mobility Services, s. r. o.

### PR DESCRIPTION
**Pull request by Richardds**
Hello,
another company suggestion.

Cheers,
Richard

### CODE REVIEW

File: companies/slovnaftbajk.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "relevant-countries": [
+        "sk"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Slovnaft Mobility Services, s. r. o.",
+    "address": "Vlčie hrdlo 1\n82412 Bratislava\nSlovakia",
+    "phone": "+421 2 5950 5950",
+    "email": "info@slovnaftbajk.sk",
```
Comment (baltpeter) [inline companies/slovnaftbajk.json:12]: As the email is not specific to data protection, we can't set the `suggested-transport-medium` to `email`. You can just remove that line.

File: companies/slovnaftbajk.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "slug": "slovnaftbajk",
+    "relevant-countries": [
+        "sk"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "Slovnaft Mobility Services, s. r. o.",
+    "address": "Vlčie hrdlo 1\n82412 Bratislava\nSlovakia",
+    "phone": "+421 2 5950 5950",
```
Comment (baltpeter) [inline companies/slovnaftbajk.json:11]: I didn't find the phone number in the privacy policy. Please add https://slovnaftbajk.sk/en/contact-us.html to the sources.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks again! Just two minor comments below. :)

File: companies/slovnaftbajk.json (hunk 1 above)
Comment (Richardds) [reply companies/slovnaftbajk.json:12]: removed

File: companies/slovnaftbajk.json (hunk 2 above)
Comment (Richardds) [reply companies/slovnaftbajk.json:11]: source added

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**